src/reports/cashflow.cpp

```diff
@@ -69,17 +70,20 @@ void mmReportCashFlow::getStats(double& tInitialBalance, std::vector<ValueTrio>&
             if (wxNOT_FOUND == accountArray_->Index(account.ACCOUNTNAME)) continue;
         }
 
-        const Model_Currency::Data* currency = Model_Account::currency(account);
-        tInitialBalance += account.INITIALBAL * currency->BASECONVRATE;
+        // Use account first transaction date for initial balance
```

Comment:
this is wrong. I have old account with initial amount not used for years - we should add account start date (date for initial amount) to use correct rate to calculate initial amount value.

Reply:
This can be separate PR as it require changes in account table, but we should think about it.